fivox/compartmentLoader.cpp

```diff
@@ -86,9 +87,10 @@ Vector2f CompartmentLoader::_getTimeRange() const
                      _impl->_report.getEndTime( ));
 }
 
-ssize_t CompartmentLoader::_load( const float time )
+ssize_t CompartmentLoader::_load( const size_t /*chunkIndex*/,
+                                  const size_t /*numChunks*/ )
 {
-    return _impl->load( time );
+    return _impl->load();
```

Comment:
I see, getNumChunks returns 1 when out-of-core is not supported. However I still think that having a load function with no arguments would be useful.